server/routes/event.py

```diff
@@ -79,6 +79,36 @@ def get_property_value(dcid: str, prop: str) -> str:
   return None
 
 
+def get_provenance(properties):
+  """
+  Returns full provenance display info for event.
+
+  Args:
+    properties: All properties of the event.
+
+  Returns:
+    A list of { url, sourceName } objects to display provenances.
+  """
+  ret = []
+
+  provenance = properties['provenance']
```

Comment:
should use properties.get('provenance') instead of accessing directly using the key incase the key doesn't exist

Reply:
good point - done